instrumentation/opentelemetry-instrumentation-remoulade/tests/test_messages.py

```diff
@@ -102,3 +102,19 @@ def test_retries(self):
             producer_spans[1], {"remoulade.retry_count": 2}
         )
         self.assertEqual(producer_spans[3].name, "remoulade/send")
+
+    def test_uninstrument(self):
+        RemouladeInstrumentor().instrument()
+
+        actor_div.send(2, 3)
+
+        spans = self.memory_exporter.get_finished_spans()
+        self.assertEqual(len(spans), 2)
```

Comment:
nit: isn't this being tested already? I think there's not much need for the instrument assertion but I leave it to you

Reply:
I want to be sure that the instrumentation works and create spans before I call uninstrument,
and after uninstrument it doesnt.
so the test covers everything